app/views/elections/show.html.erb

```diff
@@ -20,6 +20,10 @@
   <%= @election.settings %>
 </p>
 
+<% if @election.audit_trail_events.present? %>
+  <%= render("partials/audit_trail", auditable: @election) %>
```

Comment:
Made into a parital so that it is also going to be quick and easy to add to views of other models.